pyproject.toml

```diff
@@ -115,12 +116,12 @@ PyJWT = "~1.7"
 python-dateutil = "~2.8"
 tensorflow = "~2.1"
 tensorflow_hub = "~0.7"
-tensorflow-addons = ">=0.7.1"
+tensorflow-addons = "^0.7.1"
 tensorflow-estimator = "2.1.0"
 tensorflow-probability = "~0.7"
 setuptools = ">=41.0.0"
 kafka-python = "^1.4"
-ujson = "^1.35"
+ujson = ">=1.35,<3.0"
```

Comment:
what's the reason for the upper limit?

Reply:
This line was added by dependabot, and I think it makes sense because we have no idea about how ujson 3.0 will look like but we want to support both ujson 1.x and 2.x.